templates/zerver/api/incoming-webhooks-walkthrough.md

```diff
@@ -545,7 +545,7 @@ If you had used `send_and_test_stream_message`, it would have called
 
 ### Custom query parameters
 
-Custom arguments passed in URL query parameters work as expected in the webhook
+Custom parameters passed in URL query parameters work as expected in the webhook
```

Comment:
I would just make this `Customer URL query parameters work as expected...`

Reply:
Done